libs/contrib/Data/InductionRecursion/DybjerSetzer.idr

```diff
@@ -4,8 +4,6 @@
 
 module Data.InductionRecursion.DybjerSetzer
 
-%default total
-
```

Comment:
Is that deletion intentional?

Reply:
Yes unfortunately that module does not check as total anymore.